youtube_dl/extractor/zdf.py

```diff
@@ -81,6 +98,8 @@ def xml_to_format(fnode):
         'title': title,
         'description': description,
         'duration': duration,
+        'thumbnail': thumbnail,
```

Comment:
You may not specify this since you provide `thumbnails`. It will be autocalculated.